plugins/kubernetes/test/controllers/kubernetes_dashboard_controller_test.rb

```diff
@@ -36,30 +44,112 @@
 
   private
 
-  def api_response
-    response = JSON.parse(parse_json_response_file('kubernetes_pod_api_response'))
```

Comment:
Since you decided not to use template files, you might as well remove `parse_json_response_file` from `test_helper.rb` and remove the .json templates themselves from the `samples` dir.